### ISSUE TITLE
missing graceful js

### ISSUE THREAD

**Issue by Moumouls** [marc136/node-folder-hash#176]
Error using npx

```
npm WARN exec The following package was not found and will be installed: folder-hash@4.0.3
node:internal/modules/cjs/loader:988
  throw err;
  ^

Error: Cannot find module 'graceful-fs'
Require stack:
- /home/runner/.npm/_npx/c0[5](https://github.com/people-vox/main/actions/runs/3893625763/jobs/6646480252#step:3:6)[6](https://github.com/people-vox/main/actions/runs/3893625763/jobs/6646480252#step:3:7)8a2dff9[7](https://github.com/people-vox/main/actions/runs/3893625763/jobs/6646480252#step:3:8)d3d2/node_modules/folder-hash/cli.js
```

It seems to be introduced by 4.0.3

**Comment by Moumouls** [comment]
Seems to be used in `cli.js` so `graceful-js` should not be removed

**Comment by marc136** [comment]
Thank you for raising this error @Moumouls. I published a hotfix [v4.0.4](https://www.npmjs.com/package/folder-hash/v/4.0.4)
